gradio/components/__init__.py

```diff
@@ -2,8 +2,12 @@
 from gradio.components.audio import Audio
 from gradio.components.bar_plot import BarPlot
 from gradio.components.base import (
+    Column,
     Component,
+    Form,
+    FormComponent,
     IOComponent,
+    Row,
```

Comment:
It would be better for users to import `Column`, `Form`, and `Row` from `gr.layouts` instead of `gr.components`, but for the sake of backwards compatibility, I have included them here